### PR TITLE
EXPOSERS: Get Group By Id

### LINKED ISSUE #393: EXPOSERS: Get Group By Id

### PR DESCRIPTION
**Pull request by khatanboyevelbek**
Closes #393

### CODE REVIEW

**Comment by GulchexrasGithub** [comment]
@khatanboyevelbek  please provide screenshots from either Postman or Swagger to show your exposer is working.

**Comment by khatanboyevelbek** [comment]
200

Screenshot (28)

**Comment by khatanboyevelbek** [comment]
400

Screenshot (29)

File: Taarafo.Core/Controllers/GroupsController.cs [hunk 1]
```diff
@@ -75,5 +76,29 @@ public ActionResult<IQueryable<Group>> GetAllGroups()
...
+        public async ValueTask<ActionResult<Group>> GetGroupByIdAsync(Guid groupId)
+        {
+            try
+            {
+                Group retrievedGroupById = 
+                    await this.groupService.RemoveGroupByIdAsync(groupId);
+
+                return Ok(retrievedGroupById);
+            }
+            catch (GroupValidationException groupValidationException)
```
Comment (GulchexrasGithub) [inline Taarafo.Core/Controllers/GroupsController.cs:90]: `catch (GoupValidationException groupValidationException)
                when (groupValidationException.InnerException is NotFoundGroupException)
            {
                return NotFound(groupValidationException.InnerException);
            }`   How is it return Not Found error without this code?

Comment (khatanboyevelbek) [reply Taarafo.Core/Controllers/GroupsController.cs:90]: I have added this exception. You can check it out

**Comment by khatanboyevelbek** [comment]
404

Screenshot (30)

**Review by GulchexrasGithub — APPROVED** [review]

**Review by GulchexrasGithub — APPROVED** [review]

**Review by cjdutoit — APPROVED** [review]

**Pull request merged**